python/ray/serve/api.py

```diff
@@ -571,3 +577,153 @@ def delete(name: str, _blocking: bool = True):
     """
     client = get_global_client()
     client.delete_apps([name], blocking=_blocking)
+
+
+@PublicAPI(stability="alpha")
+def multiplexed(
+    func: Optional[Callable[..., Any]] = None, max_num_models_per_replica: int = -1
+):
+    """[EXPERIMENTAL] Coverts a function or method to a multiplexed function.
+
+    The function can be standalone function or a method of a class. The
+    function must have exact one argument with type `str` and the argument
+    must be the first argument. The `str` argument is the model id of the model to
+    be loaded.
+
+    User have to define the function with `async def` and the function must be
+    an async function. It is recommended to define courotine for long running
+    IO tasks in the function, it can prevent blocking the event loop.
```

Comment:
```suggestion
    an async function. It is recommended to define coroutines for long running
    IO tasks in the function to avoid blocking the event loop.
```